src/index.ts

```diff
@@ -282,6 +282,19 @@ export function getBundleId() {
   return bundleId;
 }
 
+let installerPackageName: string;
+export async function getInstallerPackageName() {
+  if (!installerPackageName) {
+    if (Platform.OS === 'android') {
+      installerPackageName = await RNDeviceInfo.getInstallerPackageName();
+    } else {
+      installerPackageName = 'unknown';
+    }
+  }
+
+  return installerPackageName;
+}
```

Comment:
Can you add the sync variant as well? In general I like them at parity unless there is something inherently async about the API making it unreasonable (like a network access) or people have petitioned for it to be constant (like some fundamental to bootstrap values)

This is in the middle area where I think both variants are best to cover everyone as best as possible without it being a constant